examples/flowc/.gitignore

```diff
@@ -3,6 +3,8 @@
 /.built
 /host
 /config.h
+/receiver
+/sender
```

Comment:
This and all of the following need to have *.exe to exclude the files in a Windows build.

Reply:
*.exe is already added in all .gitignore